src/hooks/useSystemIntakeContacts.tsx

```diff
@@ -45,31 +51,46 @@ function useSystemIntakeContacts(
   // Get system intake from Apollo cache
   const { systemIntake } = useSystemIntake(systemIntakeId);
 
-  /** Formatted system intake contacts object */
-  const contacts = useMemo<FormattedContacts | null>(() => {
-    // Get systemIntakeContacts
+  /**
+   * Formatted system intake contacts object
+   * */
+  const contacts = useMemo<FormattedContacts>(() => {
+    /** Array of system intake contacts */
     const systemIntakeContacts = data?.systemIntakeContacts
       ?.systemIntakeContacts as SystemIntakeContactProps[];
```

Comment:
I think this could be pulled up in scope outside of the useMemo, as this coalescing chain is used often throughout the component